build.sbt

```diff
@@ -92,6 +94,7 @@ lazy val copyJARToVSCode =
 lazy val `lsp-server` =
   project
     .settings(
+      name := "ralph-lsp",
```

Comment:
moving the name here because of this: https://github.com/sbt/sbt-native-packager/issues/1530

Reply:
:+1: good idea